From 5bb362d84103c88eca9064eb024d64590c99a105 Mon Sep 17 00:00:00 2001
From: Junio C Hamano <gitster@pobox.com>
Date: Fri, 9 Oct 2009 13:44:53 -0700
Subject: [PATCH 22/22] s/entire/extra/ in include-tag

--

Message-ID: <7vskdss3ei.fsf@alter.siamese.dyndns.org>

> +Capability include-tag
> +~~~~~~~~~~~~~~~~~~~~~~
> +
> +When packing an object that an annotated tag points at, include the
> +tag object too.  Clients can request this if they want to fetch
> +tags, but don't know which tags they will need until after they
> +receive the branch data.  By enabling include-tag an entire call
> +to upload-pack can be avoided.
> +

I think you are avoiding an "extra" call; you would need one entire call
to upload-pack anyway for the primary transfer.
---
 Documentation/technical/http-protocol.txt | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/Documentation/technical/http-protocol.txt b/Documentation/technical/http-protocol.txt
index 3fc8f1f..0b1c96c 100644
--- a/Documentation/technical/http-protocol.txt
+++ b/Documentation/technical/http-protocol.txt
@@ -226,12 +226,12 @@ description of the dumb server response.
 
 Smart Server Response
 ^^^^^^^^^^^^^^^^^^^^^
-If the server does not recognize the requested service name, or the
-requested service name has been disabled by the server administrator,
-the server MUST respond with the '403 Forbidden' HTTP status code.
+  If the server does not recognize the requested service name, or the
+  requested service name has been disabled by the server administrator,
+  the server MUST respond with the '403 Forbidden' HTTP status code.
 
-Otherwise, smart servers MUST respond with the smart server reply
-format for the requested service name.
+  Otherwise, smart servers MUST respond with the smart server reply
+  format for the requested service name.
 
 Cache-Control headers SHOULD be used to disable caching of the
 returned entity.
@@ -344,7 +344,7 @@ Capability include-tag
 When packing an object that an annotated tag points at, include the
 tag object too.  Clients can request this if they want to fetch
 tags, but don't know which tags they will need until after they
-receive the branch data.  By enabling include-tag an entire call
+receive the branch data.  By enabling include-tag an extra call
 to upload-pack can be avoided.
 
 Capability thin-pack
-- 
1.8.4.rc4.527.g303b16c

